### ISSUE TITLE
Custom hud element positioning/alignment

### ISSUE THREAD

**Issue by WorldOfSoap** [ImpactDevelopment/ImpactClient#1471]
Something very handy would be being able to adjust the position on screen or alignment to corners of things such as the hud info (speed, baritone, etc), coordinates, or the module list.

Additionally, being able to align things to the same corner. For example being able to configure the coordinates (bottom right by default) and hud info such as baritone (top left by default), to be all in the same corner above the module list (top right). Or all three in the bottom left corner, etc.

This would allow for far greater control over your hud / being able to customize it how you want, and would let impact play nicer with other modifications such as minimaps rather than needing to just let things overlap.

**Comment by sapioit** [comment]
Agreed. This is needed. I like to have subtitles on so I can see the nearby sounds, but that interferes with the coordinates position. I'd rather have the coordinates in the top-left and minimap in the top-right. Note: We currently only have a radar, not a fully-fledged minimap.

**Comment by slinkayy** [comment]
[There's so much empty space](https://i.imgur.com/A1OhsoT.png) to the left of the subtitles depending on your gui scale, could easily be utilized by Impact.

**Comment by SuperIsakSwahn** [comment]
i just wanna delete it and i cant find it in the files:/
